prow/jobs/test-infra/validate-scripts.jobs.yaml

```diff
@@ -0,0 +1,13 @@
+presubmits:
+  kyma-project/test-infra:
+  - name: prow/test-infra/validate-scripts
+    trigger: "(?m)^/check scripts"
+    rerun_command: "/check scripts"
+    run_if_changed: "/*.sh$"
+    decorate: true
+    skip_report: false
+    spec:
+      containers:
+        - image: eu.gcr.io/kyma-project/prow/test-infra/develop/bootstrap:v20181115-af8fb73
```

Comment:
I think we should remove `develop` directory for test-infra images. We will not have release on this repository. So this directory is useless in that case. @suleymanakbas91 what do you think?

Also how about consolidating all validation jobs in one file?

Reply:
Consolidation is done. It makes sense to remove `develop` directory but I'll do in my next PR because it needs some more changes not related to this one.

Reply:
Great :)